common/xdc/event_generator.go

```diff
@@ -26,6 +26,12 @@ import (
 	"time"
 )
 
+var (
+	defaultBatchFunc = func(batch []Vertex) bool {
+		return len(batch) == 0
+	}
+)
+
 type (
 	// Model represents a state transition graph that contains all the relationships of Vertex
 	Model interface {
```

Comment:
it will make the code cleaner if different interfaces / implementations are put in different files

Reply:
Moved to a different file